lib/mlir/Dialect/Clift/IR/CliftTypes.cpp

```diff
@@ -175,13 +175,27 @@ ArrayType::verify(::llvm::function_ref<::mlir::InFlightDiagnostic()> emitError,
   return mlir::success();
 }
 
+static mlir::clift::TypedefAttr getTypedefAttr(mlir::Type Type) {
```

Comment:
Out of curiosity, did you catch this bug while testing using the model type system tests from `revng`?

That'd be proof this wasn't a pointless exercise `:D`.

Reply:
I forget, but I think it was just something that occurred to me while working on the verifications.